src/MobiWeb/Rest/Authentication.php

```diff
@@ -108,17 +102,12 @@ public function isAuthenticated() :bool{
         if($interval->s >= Authentication::VALIDITY_PERIOD) return false;
 
         if(($interval->s < Authentication::VALIDITY_PERIOD) && ($interval->s >= (Authentication::VALIDITY_PERIOD - Authentication::VALIDITY_THRESHOLD))){
-            if(!$this->refresh()){
-                throw new \Exception("Refresh connection failed");
-                return false;
-            }
+            $this->refresh();
```

Comment:
If the refresh method is not successful, an exception will be thrown within it anyway.